README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+    Responsive websites respond to the size of the browser at any given point while adaptive websites only respond at specific points.
+
 2. Describe what it means to be mobile first vs desktop first.
 
+    Mobile-first means creating a website for mobiles first and then using min-width media queries to adapt to larger devices.  Desktop-first means creating a webtsite for desktops first and then using max-wdith media queries to adapt to smaller devices.
+
 3. What does `font-size: 62.5%` in the `html` tag do for us when using `rem` units?
 
+    It provides the font-size for the root element that the rem units reference.
+
```

Comment:
It also sets that root element to 10px, which makes calculations easier than calculating by 16px.